src/mysqlsh_helpers.py

```diff
@@ -15,18 +15,23 @@
 
 MYSQLD_SOCK_FILE = "/var/run/mysqld/mysqld.sock"
 MYSQLSH_SCRIPT_FILE = "/tmp/script.py"
+MYSQLD_CONFIG_FILE = "/etc/mysql/conf.d/z-custom.cnf"
```

Comment:
thoughts on renaming `/etc/mysql/conf.d/z-custom.cnf` -> `/etc/mysql/conf.d/z-report-host-custom.cnf` (or something similar)?

Reply:
That's very good idea, given the propensity to create other specialized